src/Components/Components.sln

```diff
@@ -236,6 +232,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Compon
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Components.Forms.Tests", "Forms\test\Microsoft.AspNetCore.Components.Forms.Tests.csproj", "{173D84A3-0F37-480F-AC0F-7E2DBBE32B28}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Samples", "Samples", "{46E4300C-5726-4108-B9A2-18BB94EB26ED}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BlazorServer.Sample", "Samples\BlazorServer.Sample\BlazorServer.Sample.csproj", "{2D7B741B-FD32-4BA6-9C98-CCC5B424B005}"
```

Comment:
Any reason for having `.Sample` in the name? It's already in a directory called `Samples`. Could this be something like `BlazorServerApp`? That would be more inline with the naming patterns for the other apps.

Reply:
Renamed to BlazorServerApp